docs/GettingStarted.md

```diff
@@ -692,6 +702,7 @@ Now that you successfully started the project, let's modify it:
 
 <block class="mac android" />
 
+- Open `index.android.js` in your text editor of choice (e.g. [Nuclide](http://nuclide.io/docs/platforms/react-native/)) and edit some lines.
 - Press the `R` key twice **OR** open the menu (F2 by default, or ⌘-M in Genymotion) and select Reload JS to see your change!
 - Run `adb logcat *:S ReactNative:V ReactNativeJS:V` in a terminal to see your app's logs
```

Comment:
Sorry, this is a more general thought. We shouldn't have people remember to write

```
adb logcat *:S ReactNative:V ReactNativeJS:V`
```

in order to see the logs. This is a pretty terrible developer experience :p